common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -113,6 +113,7 @@ const (
 		`completion_event: ?, ` +
 		`completion_event_data_encoding: ?, ` +
 		`task_list: ?, ` +
+		`is_cron: ?, ` +
```

Comment:
I don't think it is required to have is_cron persisted. This info can be retrieve with len(cron_schedule) > 0.